apps/sms/services/js/message_manager.js

```diff
@@ -216,9 +301,10 @@ var MessageManager = {
     return sendOpts;
   },
 
-  // consider splitting this method for the different use cases
-  /*
-   * `opts` can have the following properties:
+  /* This is the primary function used to send SMS for this object.
+   *
+   * TODO: Splitting this complicated method
```

Comment:
@azasypkin , this comment was present before I started working on the bug. Could I remove it?